web/src/echo.ts

```diff
@@ -272,7 +269,7 @@ export function insert_local_message(
         content_type: "text/html",
         sender_email: people.my_current_email(),
         sender_full_name: people.my_full_name(),
-        avatar_url: current_user.avatar_url,
+        avatar_url: people.small_avatar_url_for_person(current_user),
```

Comment:
Why this change? I think this might change behavior -- we were previously not using the "small" avatar URL here -- so that might be a bug fix?

Reply:
`avatar_url` can be undefined, and we want to make sure it's always defined here, which I think it always was? 

`small_avatar_url_for_person` returns the `person.avatar_url` if it is defined, but then properly sets up and returns backup values if it's not, which seemed more correct.